mne_bids/mne_bids.py

```diff
@@ -382,11 +381,6 @@ def _sidecar_json(raw, task, manufacturer, fname, kind, eeg_ref=None,
         Filename to save the sidecar json to.
     kind : str
         Type of the data as in ALLOWED_KINDS.
-    eeg_ref : str
-        Description of the type of reference used and (when applicable) of
-        location of the reference electrode.  Defaults to None.
-    eeg_gnd : str
-        Description  of the location of the ground electrode. Defaults to None.
```

Comment:
@sappelhoff I removed these two because I feel it's something that the user should do directly via a text editor, not python. 

I would say let's not aim for perfection, but if we can get 80% of the job done, `mne-bids` will be useful for most people.

Reply:
okay, fair enough for `eeg_gnd`, but `eeg_ref`(which is EEGReference in BIDS) is a REQUIRED field in the EEG BEP006, so I would keep it.

Reply:
alright done